bin/node-template/node/src/command.rs

```diff
@@ -98,16 +99,21 @@ pub fn run() -> sc_cli::Result<()> {
 				Ok((cmd.run(client, backend, None), task_manager))
 			})
 		},
-		Some(Subcommand::Benchmark(cmd)) =>
-			if cfg!(feature = "runtime-benchmarks") {
-				let runner = cli.create_runner(cmd)?;
+		Some(Subcommand::Benchmark(cmd)) => {
+			let runner = cli.create_runner(cmd)?;
 
-				runner.sync_run(|config| cmd.run::<Block, service::ExecutorDispatch>(config))
-			} else {
-				Err("Benchmarking wasn't enabled when building the node. You can enable it with \
-				     `--features runtime-benchmarks`."
-					.into())
-			},
+			runner.sync_run(|config| match cmd {
+				BenchmarkCmd::Pallet(cmd) =>
+					if cfg!(feature = "runtime-benchmarks") {
+						cmd.run::<Block, service::ExecutorDispatch>(config)
+					} else {
+						Err("Runtime benchmarking wasn't enabled when building the node. \
+							You can enable it with `--features runtime-benchmarks`."
+							.into())
+					},
+				_ => Err("Sub-command is not supported".into()),
+			})
+		},
```

Comment:
Why not add the other sub commands to the node template? Seems like they would be helpful if someone is basing a new project off the template

Reply:
Yea that's a good follow-up, I'll put it in an issue. Just wanted to get it done faster :smile:  

The node-template currently uses the weights from `frame_support`, so even when you run the benchmarks yourself, it would require some updating to actually use them in the node-template.

Reply:
Okay I added it now in the [last commit](https://github.com/paritytech/substrate/pull/11164/commits/e2201af5d7a89baf00bd5429897f361ef265f370) since @shawntabrizi also wanted it.  
There is infrastructure missing in the node-template, therefore the copy&paste from the substrate node.